lib/modules/showImages.js

```diff
@@ -786,6 +786,9 @@ async function completeExpando(expando, thing, mediaInfo) {
 
 	Object.assign(expando, options);
 
+	const hideButton = thing.getHideElement();
```

Comment:
Note: this adds a listener to both the hide and unhide buttons, but this is correct because the unhide button also removes the post from the page (the /u/me/hidden page, the only place it shows up)